driver/src/price_finding/linear_optimization_price_finder.rs

```diff
@@ -43,11 +52,12 @@ fn serialize_order(order: &models::Order, id: &str) -> serde_json::Value {
     })
 }
 
-fn serialize_balances(balances: &Vec<u128>) -> serde_json::Value {
+fn serialize_balances(balances: &Vec<u128>, num_tokens: u8) -> serde_json::Value {
```

Comment:
Shouldn't this function also have something to do with `num_accounts`?

Reply:
We don't actually store num_accounts in our system yet. It's assumed that everything we get from the database is a multiple of `TOKENS`, that multiple being num_accounts.

Reply:
I believe num_accounts is stored as part of the [SnappInitialization event](https://github.com/gnosis/dex-contracts/blob/4df0570affbe7de86207f790a3a2f8c7f1ccf07b/contracts/SnappBase.sol#L56)

Reply:
Yeah, currently, we are not using these values from the db at all. Also models::Tokens is also just a constant. I would be in favour of making a seperate issue for this and merging this PR

Reply:
#86

Reply:
Cool beans!